tests/h/views/help_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+import mock
+from pyramid import httpexceptions
+
+from h.views import help
+
+
+def test_welcome_page_redirects_to_new_url(pyramid_request, routes):
```

Comment:
Typically if you don't actually use a fixture within the test, you should use the `pytest.mark.usefixtures` decorator rather than passing it in, e.g.

```python
@pytest.mark.usefixtures('routes')
def test_welcome_page_redirects_to_new_url(pyramid_request):
    ...
```